src/components/common/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx

```diff
@@ -148,9 +150,7 @@ const ManageReputationDialogForm = ({
     ['value'],
   );
 
-  const domainName = colonyDomains.find(
-    (domain) => domain?.nativeId === domainId,
-  )?.metadata?.name;
+  const domainName = findDomainByNativeId(domainId, colony)?.metadata?.name;
```

Comment:
I'm really not happy with this approach. 

Either destructure it, or use an intermediate variable